pkg/cli/build/build.go

```diff
@@ -66,6 +66,11 @@ func Run(*cli.Context) error {
 		return nil
 	}
 
+	if err = downloadKubernetesSELinuxPolicies(ctx); err != nil {
+		audit.AuditInfof("Configuring Kubernetes failed. %s", checkLogMessage)
```

Comment:
When I added `AuditInfof`, I had an iteration where there was an `AuditErrorf`. I removed it because I didn't have a use for it, but this feels like a use (and probably not the last).

How would you feel about adding in `AuditErrorf` and for now simply chaining it back to `AuditInfof` under the covers? That way if we do decide we want to output error information differently (*cough* in red *cough*), we can simply update that implementation and it'll be consistent across the entire CLI.

Reply:
I don't think we want to have the error itself in the audits. I'm pretty sure we set on having any Go errors be logged to the file and only redirect the user there since their formatting can vary wildly and may lead to a bad UX. This is why `AuditErrorf` seems like a bad idea to me but I'm open to suggestions.

Reply:
It's not worth speculating too much right now. Let's see what the next few weeks of testing bring up in terms of UX (I suspect we'll need some significant changes across the board) and then this may be a moot point.